app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<!-- This is the place for the content of the user page -->
```

Comment:
- [ ] Please, as stated in the requirements, let's ensure that all view files have some placeholders written in plain HTML. :ok_hand:  

### Requirement: :point_down: 

![Screenshot from 2023-12-05 23-43-38](https://github.com/maximoortelli/Blog_App/assets/89228743/073b4420-1e7f-4bd2-9040-2ab2504505ca)